doc/source/library_description/packaging.rst

```diff
@@ -77,7 +97,7 @@ Verify the distribution's long description rendering with ``twine``.
   pip install twine
   twine check dist/*
 
-Upload the package to PyPI using ``twine`` and the upload token generated for the ``ansys`` PyPI account. 
+Upload the package to PyPI using ``twine`` and the upload token generated for the ``ansys`` PyPI account.
```

Comment:
@greschd  We can add that an independent token dedicated to the package can then be created as soon as the package is available on Pypi. This way the token stored in the GitHub secrets and used in the release workflow is only related to that specific package.
```suggestion
Upload the package to PyPI using ``twine`` and the upload token generated for the ``ansys`` PyPI account.
As soon as the package the package has been released for the first time, it is possible to create an independent token dedicated to this package.
This way the token stored in the GitHub secrets and used in the release's workflow is only related to that specific package. This minimize the exposure of any potential token security flaws.
```

Reply:
Agreed. Nice work! Specify the default name for the org secret and this will be good to add.